source/synthDrivers/espeak.py

```diff
@@ -50,6 +51,14 @@ class SynthDriver(SynthDriver):
 	}
 	supportedNotifications = {synthIndexReached, synthDoneSpeaking}
 
+	# A mapping of commonly used language codes to eSpeak languages.
+	# Introduced due to eSpeak issue: https://github.com/espeak-ng/espeak-ng/issues/1200
+	# These are used when eSpeak doesn't support a given language code
+	# but a default alias is appropriate
+	_defaultLangToLocale = {
+		"en": "en-gb",

```

Comment:
expand this list to include "fr" and "chr", the languages in 
`set(lang.split("-")[0] for lang in self.availableLanguages).difference(self.availableLanguages)`